### DISCUSSION TITLE
Building from a Mac, works on Parallels Desktop?

### DISCUSSION THREAD

**Discussion by soymonitus** [SynthstromAudible/DelugeFirmware#22] [Q&A]
I work with a Macbook, and have Parallels Desktop with Windows 7 and Windows XP. Will the dev environment work on those virtualized systems? Which is the preferred one: XP or 7 or other?

**Comment by litui** [comment]
I have tested neither of those two versions of windows. The more important question is whether Parallels Desktop is running as 64bit as the build tools default to 64 bit versions.

**Comment by litui** [comment]
I should note as well, that native mac support is a work in progress that should be ready in the next week or so. I'm working on build tools/scripts that will support Windows, Mac, and Linux on x86_64 platforms.

**Reply by weavermedia** [reply]
Is there any process on building in macOS? I wish I could help the effort but I'm pretty sure building the tools that build the code is a step or two beyond me 😜

**Reply by litui** [reply]
Oh it works now. Run ./dbt in a terminal and all will be revealed. =)

**Reply by weavermedia** [reply]
Excellent! 👏

**Comment by jamiefaye** [comment]
"Deluge Build Tool"?

**Reply by litui** [reply]
~~"Dialectical Behavioural Therapy"~~ oh, wrong website. I mean "Deluge Build Tool", @jamiefaye's right =P
